Gemfile

```diff
@@ -37,6 +37,7 @@ gem 'aasm', '~> 4.12.0'
 
 gem 'rubyXL'
 gem 'ruby-progressbar'
+gem 'tzinfo-data'
```

Comment:
what is this?

Reply:
When I tried to run the application a while back, it notified me of an error that I need this gem in the Gemfile.

Reply:
I'm fine with removing the sqlite line above but as far as I can tell, this gem is not needed, so please remove this.